src/system/esp32/network.c

```diff
@@ -12,271 +12,236 @@
  *   ADLINK zenoh team, <zenoh@adlink-labs.tech>
  */
 
-#include <lwip/sockets.h>
-#include <errno.h>
 #include <netdb.h>
-#include <net/if.h>
-#include <unistd.h>
+
 #include "zenoh-pico/system/common.h"
 #include "zenoh-pico/utils/private/logging.h"
 
-/*------------------ Interfaces and sockets ------------------*/
-char *_zn_select_scout_iface()
+/*------------------ Endpoint ------------------*/
+void _zn_release_tcp_endpoint(void *arg)
```

Comment:
`_zn_release_tcp_endpoint` => `_zn_release_endpoint_tcp`

Reply:
Agree and done in the next commit.